pkg/hosts/privdrop_unix_test.go

```
@@ -0,0 +1,236 @@
+	"syscall"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestNeedsPrivilegeDrop(t *testing.T) {
+	t.Parallel()
+
+	tests := []struct {
+		name       string
```

golangci-lint (2.12.2)

[error] File is not properly formatted

(gofmt)